docker-compose.yml

```diff
@@ -6,38 +6,56 @@ services:
         command: >
             /bin/bash -c "
                 until echo > /dev/tcp/postgres/5432; do sleep 1; done
-                npx pubsweet server
+                node app
             "
         ports:
             - 3000:3000
         depends_on:
             - postgres
+        env_file:
+          - .env
         environment:
-            NODE_ENV: development
+            NODE_ENV: production
             NODE_CONFIG: '{
-                "pubsweet-client": {
-                    "API_ENDPOINT": "/api"
-                },
-                "pubsweet-server": {
-                    "baseUrl": "${PUBSWEET_BASEURL}",
```

Comment:
if these variables become passed directly, should do a whitelist with just their names, details in https://github.com/elifesciences/elife-xpub-deployment/pull/10/files#r219819293